tests/conftest.py

```diff
@@ -82,7 +82,7 @@ def tmp_dir():
     os.chdir(newpath)
     yield
     os.chdir(cwd)
-    shutil.rmtree(newpath)
+    shutil.rmtree(newpath, ignore_errors=True)
```

Comment:
Request adding comment to explain why ignore_errors is needed e.g. `# ignore_errors for windows developement`